server/data/data-cassandra/src/main/java/org/apache/james/droplists/cassandra/CassandraDropListModule.java

```diff
@@ -0,0 +1,47 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.droplists.cassandra;
+
+import org.apache.james.backends.cassandra.components.CassandraModule;
+import org.apache.james.droplists.cassandra.tables.CassandraDomainDropListTable;
+import org.apache.james.droplists.cassandra.tables.CassandraGlobalDropListTable;
+import org.apache.james.droplists.cassandra.tables.CassandraUserDropListTable;
+
+import com.datastax.oss.driver.api.core.type.DataTypes;
+
+public interface CassandraDropListModule {
+    CassandraModule MODULE = CassandraModule.table(CassandraGlobalDropListTable.TABLE_NAME)
```

Comment:
Hello! I was guided by the fact that adding a column like DROP_LIST_TYPE would lead to the need for additional filtering when making a selection. This will degrade performance, won't it?

Reply:
I don't think that table will be that big honestly, should not change much perf wise.

However, I know that having too much tables in Cassandra affects performance as well, if I remember correctly. mainly why we did a refactoring on quotas to reduce the number of tables used for example

But @chibenwa is the Cassandra expert, maybe extra insight welcomed?